.all-contributorsrc

```diff
@@ -294,6 +294,12 @@
         "translation"
       ]
     },
+    {
+      "login": "tphbrok",
+      "name": "Thomas Brok",
+      "avatar_url": "https://avatars0.githubusercontent.com/u/11331876?v=4",
+      "profile": "https://github.com/tphbrok",
```

Comment:
No `contributions`

Reply:
Oh! Well spotted.

Reply:
also not see the badge's number updated.

Reply:
It's been updated in 77b165228e3a632c8f49ca6a89250592b8a5c37d which is still effective on the latest state of that branch (considering it updated to 31 and the README.md is at 31).
Not sure why it's not showing up on the "Files changed" tab tho when none of the later commits undid that.